pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.tsx

```diff
@@ -216,9 +222,16 @@ export const Table: FC<TableProps> = ({
           ) : null}
           {rows.length && !loading ? (
             <div {...getTableProps()} className="table">
-              <div {...getTableBodyProps()} className={cx('table-body', styles.tableBody(scroll.y))}>
-                {rows.map(RenderRow)}
+              <div
+                {...getTableBodyProps()}
+                className={cx('table-body')}
+                style={{ height: scroll.y - 70 }}
+              >
+                <Scrollbar forceVisible="x" style={{ maxHeight: scroll.y - 70 }}>
```

Comment:
I think you should consider using a constant here instead of the hardcoded 70

Reply:
Good point, thanks!